Models/Settings/Presets.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace MMRando.Models
+{
+    public enum Presets
+    {
+        Default,
+        Casual,
+        FullRando,
+        Accessible,
```

Comment:
Since the Accessible setting will be removed, I feel like this whole Presets list is unnecessary. Full Rando is not hard to set up. I don't know exactly what the Casual preset is, but it seems kinda subjective to me.